### PR TITLE
Fix action codes for outright possession order

### PR DESCRIPTION
**Pull request by maysakanoni**
**What?**

- Fix the action codes required for an outright possession 

**Why?** 

- The incorrect action codes were being used

### CODE REVIEW

File: lib/hackney/tenancy/court_outcome_codes.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module Hackney
+  module Tenancy
+    module CourtOutcomeCodes
```
Comment (richrace) [inline lib/hackney/tenancy/court_outcome_codes.rb]: 👍

File: spec/lib/hackney/income/tenancy_prioritiser/classifications/apply_for_outright_possession_warrant_spec.rb [hunk 2]
```diff
@@ -5,10 +5,15 @@
...
     active_agreement: false,
     courtdate: 1.month.ago,
-    court_outcome: 'OUT'
+    court_outcome: 'OPD'
   }
   examples = [
     base_example,
+    base_example.merge(
+      desription: 'when the court outcome is outright possession forthright',
+      outcome: :apply_for_outright_possession_warrant,
```
Comment (richrace) [inline spec/lib/hackney/income/tenancy_prioritiser/classifications/apply_for_outright_possession_warrant_spec.rb]: I don't think we really need this line

File: lib/hackney/income/tenancy_prioritiser/tenancy_classification.rb [hunk 3]
```diff
@@ -51,7 +51,7 @@ def apply_for_outright_possession_warrant?
           return false if @criteria.courtdate.blank?
           return false if @criteria.courtdate.future?
           return false if @criteria.courtdate < 3.months.ago
-          @criteria.court_outcome == Hackney::Tenancy::ActionCodes::OUTRIGHT_POSSESSION_ORDER
```
Comment (richrace) [inline lib/hackney/income/tenancy_prioritiser/tenancy_classification.rb]: Have you removed `Hackney::Tenancy::ActionCodes::OUTRIGHT_POSSESSION_ORDER` or is it still required?

File: spec/lib/hackney/income/tenancy_prioritiser/classifications/apply_for_outright_possession_warrant_spec.rb (hunk 2 above)
Comment (maysakanoni) [reply spec/lib/hackney/income/tenancy_prioritiser/classifications/apply_for_outright_possession_warrant_spec.rb]: I agree, I've removed this line

**Pull request merged**